src/MLJClusteringInterface.jl

```diff
@@ -26,7 +26,7 @@ const Cl = Clustering
 
 # Definitions of model descriptions for use in model doc-strings.
 const KMeansDescription ="""
-K-Means algorithm: find K centroids corresponding to K clusters in the data.   
+K-Means algorithm: find K centroids corresponding to K clusters in the data.
```

Comment:
You can delete these "Description" constants, as you are rendering them redundant. Remember to delete the corresponding `descr=...` declarations in the `metadata_model` declarations